HOWTO-nl.md

```diff
@@ -0,0 +1,17 @@
+Lees dit in andere talen [English](HOWTO.md), [Français](HOWTO-fr.md), [Español](HOWTO-es.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md), [Русский](HOWTO-ru.md).
+
+Welkom bij Free-Programming-Books! We verwelkomen nieuwe bijdragers; zelfs degenen die hun allereerste pull-verzoeken doen. Als je een van hen bent, zijn hier enkele bronnen die je kunnen helpen:
+
+* [Over pull-verzoeken](https://help.github.com/articles/about-pull-requests/)
+* [Een pull-verzoek maken](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
+* [Github Hallo Wereld](https://guides.github.com/activities/hello-world/)
+* [Youtube - Github-zelfstudie voor beginners](https://www.youtube.com/watch?v=0fKg7e37bQE)
+* [Youtube - Hoe een GitHub-repo te forken en een pull-verzoek in te dienen](https://www.youtube.com/watch?v=G1I3HF4YWEw)
+* [Youtube - Markdown Crash Course](https://www.youtube.com/watch?v=HUBNt18RFbo)
```

Comment:
Take into account to add a note according to this links are in english. Translate `in english`

Example:
```markdown
* [Over pull-verzoeken](https://help.github.com/articles/about-pull-requests/) *(in english)*
```